Zend/tests/bug37676/7.phpt

```diff
@@ -0,0 +1,9 @@
+--TEST--
+Bug #37676 (Constant warnings)
+--FILE--
+<?php
+const FOO = null;
+const BAR = FOO[0][1];
+?>
+--EXPECTF--
+Warning: Variable of type null does not accept array offsets in %s on line %d
```

Comment:
Looks like this test is segfaulting on Travis.

Reply:
I am unsure why this is happening on travis.  I ran with valgrind on my mac, and all tests LEAK&FAIL, not just for my bug37676 directory (may be problems with git version of valgrind for 10.13) . I did detect a leak in linux valgrind, which I pushed.